meta/Meta.cpp

```diff
@@ -1856,6 +1856,37 @@ sai_status_t Meta::objectTypeGetAvailability(
     return status;
 }
 
+sai_status_t Meta::queryAttributeCapability(
+        _In_ sai_object_id_t switchId,
+        _In_ sai_object_type_t objectType,
+        _In_ sai_attr_id_t attrId,
+        _Out_ sai_attr_capability_t *capability)
+{
+    SWSS_LOG_ENTER();
+
+    PARAMETER_CHECK_OID_OBJECT_TYPE(switchId, SAI_OBJECT_TYPE_SWITCH);
+    PARAMETER_CHECK_OID_EXISTS(switchId, SAI_OBJECT_TYPE_SWITCH);
+    PARAMETER_CHECK_OBJECT_TYPE_VALID(objectType);
+
+    auto mdp = sai_metadata_get_attr_metadata(objectType, attrId);
```

Comment:
What is `mdp`? I would make the name more specific.

Reply:
I think it stands for meta data pointer. I am merely follow the convention setup in this file with all previous existing functions that make the same call with same naming.